modules/ovis/samples/ovis_anim_demo.cpp

```diff
@@ -0,0 +1,46 @@
+#include <opencv2/highgui.hpp>
+#include <opencv2/calib3d.hpp>
+
+#include <opencv2/ovis.hpp>
+
+#include <iostream>
+
+
+#define KEY_ESCAPE 27
+
+using namespace cv;
+
+int main()
+{
+  const double time_delta = 0.1;
+
+  const Size2i imsize(800, 600);
+  const double focal_length = 800.0;
+
+  // random calibration data, your mileage may vary
+  Mat1d cm = Mat1d::zeros(3, 3); // init empty matrix
+  cm.at<double>(0, 0) = focal_length; // f_x
+  cm.at<double>(1, 1) = focal_length; // f_y
+  cm.at<double>(2, 2) = 1; // f_z
+  Mat K = getDefaultNewCameraMatrix(cm, imsize, true);
+
+  ovis::addResourceLocation("packs/Sinbad.zip"); // shipped with Ogre
+
+  Ptr<ovis::WindowScene> win = ovis::createWindow(String("ovis"), imsize, ovis::SCENE_INTERACTIVE | ovis::SCENE_AA);
+  win->createEntity("sinbad", "Sinbad.mesh", Vec3i(0, 0, 0), Vec3f(0, 0, 0));
+  win->setCameraPose(Vec3f(0, 0, 20));
+  win->setCameraLookAt("sinbad");
+  win->createLightEntity("sun", Vec3i(0, 0, 100));
+
+  // play idle animation
+  win->setEntityAnimation("sinbad", "IdleBase");
```

Comment:
instead of creating a whole new sample, copying the existing one, just add the animation to the ovis_demo.{py, cpp}

Reply:
I'd also like to have just one example. However, I did want to showcase the animation playback and the updated demo allows to list and play all available animations on the _Sinbad_ mesh.

Please tell me if it still makes sense to merge both demos.